canonical-web-and-design/vanilla-framework
# Rename "p-label" to "p-status-label"

## Done

- Create a new component: `p-status-label` and add it to the examples section
- Rename "Label"  references to "Status label"

(I'm happy to use the deprecate mixin but I couldn't find an example)

Fixes #4395 

## QA

- Open demo
- Go to `/docs/examples` and check there are both a "label" and a "status label" components (and both work)
- Review updated documentation:
  - "Label" should not appear in the side nav
  - Instead, there should be a "Status labels" pattern
  
### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical-web-and-design/vanilla-framework/wiki/Release-process#pre-release-tasks)
